hr_timesheet_sheet_autodraft/tests/test_hr_timesheet_sheet_autodraft.py

```diff
@@ -27,6 +27,9 @@ def test_no_autocreate_by_default(self):
         )
         employee = self.HrEmployee.create({"name": "Employee", "user_id": user.id})
         project = self.Project.create({"name": "Project"})
+
+        self.company_id.timesheet_sheets_autodraft = False
```

Comment:
But that should be the default, no?

Reply:
Well, that's the point of `test_no_autocreate_by_default` test? That by default, w/o setting the value, it's not autocreated?

Reply:
It can be created i'll remove it.